sdk/hack/release.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+TAG_NAME=$1
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
+
+if [[ -z "$TAG_NAME" ]]; then
+  echo "Usage: release.sh <release-tag>" >&2
+  exit 1
+fi
+
+sed -i.bak -e "s|__version__ =.\+|__version__ = '$TAG_NAME'|g" "$DIR/../python/kfp/__init__.py"
```

Comment:
Since it's already a script, not Python, here how about just use the root /VERSION file?
Or is it because you want to split the versioning that SDK has different version with BE/UI? I prefer same branch has same version.

Reply:
The only reason is to make the script self-contained within the folder. It doesn't need to know external structure. When being called from `root/hack/release-imp.sh`, the entire repo will have the same version.